### DISCUSSION TITLE
Support Chinese for Kyuubi

### DISCUSSION THREAD

**Discussion by zhenjiaguo** [apache/kyuubi#2442] [General]
A lot of open source projects that support both English and Chinese, such as [Flink](https://flink.apache.org/zh/) and [Doris](https://doris.apache.org/zh-CN/).

In order to grow Apache Kyuubi community, I think we should support Chinese languages.

What do you think?

**Comment by yaooqinn** [comment]
good to have a CN version doc

**Reply by zhenjiaguo** [reply]
OK, I will work on this to make kyuubi better.

**Reply by yaooqinn** [reply]
thanks @zhenjiaguo
